### PR TITLE
Fix #970: Add an XML Linter to the project [SAMPLE PR]

### LINKED ISSUE #970: Add an XML Linter to the project
Our project has a lot of XML files that describe the look and feel of the application. We would like to add a linter that ensures all the XML files follow the same style.
[Xmllint](http://xmlsoft.org/xmllint.html) seems to be a good fit for us and we would like to use that for our XML files. If you have recommendations for some other linter, please reach out to me to discuss if its a viable option before using it for the project. 
Steps that need to be completed to add the linter:

- [ ] Run the linter on our existing XML files. Create a PR that fixes all lint problems that the linter catches

- [ ] Add steps in the linter job of our CircleCI workflow (similar to the [Kotlin linting](https://github.com/oppia/oppia-android/blob/3e1c0a8fdf5a575e23eb762725e9f11a228e22a4/.circleci/config.yml#L63) step) that will install and run the linter on our XML files

- [ ] Verify that the Android Studio auto-formatter is using the same rules as the linter configuration. If not, make changes to the code style of the XML files in Android Studio (that will create changes to the files present in the .idea/codeStyles/ directory). Create a PR that checks in the changes in these files.

- [ ] Add documentation on how to install the linter in the [prerequisites section of our wiki](https://github.com/oppia/oppia-android/wiki#prerequisites)

### PR DESCRIPTION
**Pull request by anandwana001**
## Explanation

Fix #970 

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

**Review by vinitamurthi — COMMENTED** [review]
Lets call the name of the file xml_linter, or otherwise bring all the linters into this yml

**Comment by anandwana001** [comment]
> Lets call the name of the file xml_linter, or otherwise bring all the linters into this yml

as per the GitHub's super linter, the file should be named as linter.yml and it will trigger automatically when push, but this doesn't seems to triggered in our case.

**Comment by anandwana001** [comment]
GitHub super linter not catching things like `UnusedAttribute`, `FragmentTagUsage` or anything which an Android lint tool catches.

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -44,6 +44,17 @@ jobs:
...
     -  name: Protobuf lint check
        run: ./buf check lint --input=model/src/main/proto --input-config buf.yaml && echo "Protobuf lint check completed successfully"
+
+    - name: Set up JDK 1.9
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.9
+
+    - name: XML lint check
+      run: ./gradlew lintDebug
```
Comment (anandwana001) [inline .github/workflows/main.yml]: is this worth trying, as we are migrating to Bazel?
@BenHenning

Comment (anandwana001) [reply .github/workflows/main.yml]: Is might help in covering Android Lint as per the issue here -  https://github.com/oppia/oppia-android/issues/1742

Comment (BenHenning) [reply .github/workflows/main.yml]: Hmm, will this work if Gradle isn't defined anymore? What does Gradle call through to--could we call that ourselves such that we don't depend on Gradle?

Comment (anandwana001) [reply .github/workflows/main.yml]: Android Lint without gradle is something I need to look. As of now I didn't see any other solution on running android lint without gradle.

**Pull request closed without merging**